client/scss/elements/_typography.scss

```diff
@@ -84,7 +84,7 @@ dl {
 }
 
 dt {
-    color: darken($color-grey-3, 15%);
+    color: $color-grey-3;
```

Comment:
From what I can see this new shade of grey is only used here. This really doesn't feel worth the extra color – would be fine to just go with `$color-grey-2`.